src/site/markdown/index.md

```diff
@@ -1,3 +1,48 @@
-# Documentation
+# TableJoin Java Client
+The TableJoin utility is designed for joining tables expressed in CSV or structed as list of String lists. 
```

Comment:
The TableJoin utility is designed for joining tables expressed in CSV or **structured** as list of String lists. 
It enhances the flexibility of data processing by allowing developers to perform joint operations on specified tables.